src/types/types.ts

```diff
@@ -865,6 +865,10 @@ export interface EdgeSignMessageOptions {
   otherParams?: JsonObject
 }
 
+export interface EdgeSignBytesOptions {
+  otherParams?: JsonObject
+}
```

Comment:
This type is unused. In the type definitions, you use `EdgeSignMessageOptions`. So maybe we can just delete it.